packages/astro/src/integration/index.ts

```diff
@@ -48,6 +48,9 @@ export const sentryAstro = (options: SentryOptions = {}): AstroIntegration => {
                   sourcemaps: {
                     assets: uploadOptions.assets ?? [getSourcemapsAssetsGlob(config)],
                   },
+                  bundleSizeOptimizations: {
+                    excludePerformanceMonitoring: options.bundleSizeOptimizations?.excludeTracing || false,
```

Comment:
m: What about passing all the bundle size optimizations through (or more specifically, defining all of them)?

Reply:
Sound good to me, @Lms24 what do you think?

Reply:
sounds good, let's do it!